### PR TITLE
Allow the CLI to find tasks by org name

### LINKED ISSUE #11781: CLI: task find runs only accepts org-id but should accept org name

### PR DESCRIPTION
**Pull request by lyondhill**
Closes #11781

### CODE REVIEW

**Comment by docmerlin** [comment]
Does this need a change to the swagger spec?

**Comment by lyondhill** [comment]
It shouldn't require a swagger update because the api should not have changed

File: task/platform_adapter.go [hunk 1]
```diff
@@ -417,6 +430,7 @@ func (p *pAdapter) populateOrg(ctx context.Context, org *platform.Organization)
 		org.Name = o.Name
 	} else {
 		o, err := p.orgSvc.FindOrganization(ctx, platform.OrganizationFilter{Name: &org.Name})
+		fmt.Println(o, err, "BLARG")
```
Comment (tkyocum) [inline task/platform_adapter.go]: Looks like this slipped past review.

File: task/platform_adapter.go [hunk 2]
```diff
@@ -51,9 +51,22 @@ func (p pAdapter) FindTaskByID(ctx context.Context, id platform.ID) (*platform.T
...
-		params.Org = *filter.Organization
+	org := platform.Organization{
+		Name: filter.Organization,
 	}
+	if filter.OrganizationID != nil {
+		org.ID = *filter.OrganizationID
+	}
+	if filter.Organization != "" {
+		if filter.Organization == "non-existant-org" {
+			fmt.Println("here??")
```
Comment (tkyocum) [inline task/platform_adapter.go]: This slipped past review, too.

**Comment by lyondhill** [comment]
Mark caught these here
On Sat, Mar 2, 2019, 12:14 PMhttps://
github.com/influxdata/influxdb/pull/12314 thanks for finding my mistake.

**Pull request merged**